packages/nextjs-demo/src/app/nl-gh-search/page.tsx

```diff
@@ -18,17 +19,17 @@ export default function Home(props: { searchParams: any }) {
         title={`Natural Language GitHub Search for: "${githubTopic}"`}
         description="AI has been asked to output prose"
       >
-        <AIComponent>
+        <AI.jsx>
           <NaturalLanguageGitHubSearch query={githubTopic} outputFormat="prose" />
-        </AIComponent>
+        </AI.jsx>
       </ResultContainer>
       <ResultContainer
         title={`Natural Language GitHub Search for: "${githubTopic}"`}
-        description="AI has been asked to output HTML, and we write it directly into the DOM. (If the AI generates something other than HTML, this could produce funky results.)"
```

Comment:
Why did you take this out? :smile:

Reply:
Oops...because at the time the innerHTML'ing didn't work. But now it does, so back it goes.